docs/photoniq/vwrs/troubleshoot-vwrs.md

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_position: 120
+title: Troubleshoot VWRs
+---
+
+This page provides troubleshooting information for Macrometa PhotonIQ Virtual Waiting Rooms (VWRs).
+
+## No Waiting Room Configured
+
+If a waiting room has not been configured, then the waiting room library returns the header _No-Waiting-Room_, which indicates no waiting room was configured. This means that the domain URL is not configured correctly.
+
+## Troubleshoot Waiting Room HTML
+
+You can configure the cloud origin (e.g. NetStorage) path to the waiting room HTML through the library or the domain record. To preview this HTML, open your preferred browser and navigate to the following URL: `https://<hostname>/<waiting-room-path>/preview`
+(for example, `http://show.store.com/checkout/preview`).  
+
+## Troubleshoot Waiting Room Cookie
+
+The _vwrs-cookie_ cookie holds valuable information regarding each user's access to the waiting room. This information is securely encrypted using the library encryption key.
```

Comment:
@rahul-naik1 would it make sense to mention what the variable for the encryption key?  But way, how is this now set using the npm package?

Reply:
@joubert-macrometa I am not sure. I will check with either @dwivedianurag or @naimishvora-macrometa for this.

Reply:
Nevermind.  Don't reference encryption key.

Reply:
Do I need to remove it?